### ISSUE TITLE
add self-signed certs to vagrant install

### ISSUE THREAD

**Issue by btopro** [elmsln/elmsln#936]
it would be nice during install for vagrant to issue it's own self-signed certs. By doing this we could use https on localhost which has a number of benefits (security, meh). For one thing this would allow us to install using https as the default (and most expected production) setup. It also would help ensure bakery works the same way in production as it does in vagrant (for most people).

It also ensures that systems that prioritize certs (like annyang) don't nag with questions about authorization to function bc of the perceived insecure connection.

https://www.sslshopper.com/article-how-to-create-and-install-an-apache-self-signed-certificate.html

**Comment by bradallenfisher** [comment]
starting this.

**Comment by bradallenfisher** [comment]
any preferences?

**Comment by btopro** [comment]
nope, just something so we can work in https locally; less for security side and more so to reduce variables for issues since the majority of deployments always forever (especially after #661 gets in there in a totally automated manner) will be cert based

**Comment by bradallenfisher** [comment]
closing in favor of #1149
